db/migrate/20190715144518_create_backend_apis.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CreateBackendApis < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def up
+    create_table :backend_apis do |table|
+      table.string :name, null: false, limit: 511
+      table.string :system_name, unique: true, null: false
+      table.text :description
+      table.string :private_endpoint
+
+      table.belongs_to :account, limit: 8
+
+      table.timestamps null: false
+      table.integer :tenant_id, limit: 8
```

Comment:
We're missing the table in https://github.com/3scale/porta/tree/master/lib/system/database/definitions